core/src/Dscp/Core/Foundation/Witness.hs

```diff
@@ -254,7 +254,7 @@ unGTxWitnessed (GPublicationTxWitnessed tw) = GPublicationTx (ptwTx tw)
 
 -- | Transaction with reference to block it is published in
 data GTxInBlock = GTxInBlock
-    { tbHeaderHash :: Maybe HeaderHash
+    { tbBlock :: Maybe Block
```

Comment:
Please explain this change. Also why is it in `core/../Witness.hs`?

Reply:
When I added this type, I only returned hash of block header for transaction in API, but now I need more info about block.
Not sure, whether it belongs to `core`, should I create something like `Dscp.Witness.Types`?